### PR TITLE
refine privacy policy and conditionally adapt for nine-default onboarding

### PR DESCRIPTION
**Pull request by hpk42**
The privacy policy now has a clear summary, noting that it might not even be neccessary to provide one but we do it anyway.

### CODE REVIEW

File: www/src/index.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
 
 <img class="banner" src="collage-top.png"/>
 
-## Dear [Delta Chat](https://get.delta.chat) users and newcomers, 
+## Dear [Delta Chat](https://get.delta.chat) users and newcomers ... 
 
-Welcome to instant, interoperable and [privacy-preserving](privacy.html) messaging :) 
+{% if config.mail_domain != "nine.testrun.org" %}
```
Comment (missytake) [inline www/src/index.md:6]:
```suggestion
{% if config.mail_domain != "nine.testrun.org" %}
Welcome to instant, interoperable and [privacy-preserving](privacy.html) messaging :) 
{% else %}
```
I think you meant to write the opposite?

**Review by missytake — APPROVED** [review]
I agree with the intent of this PR :) some remarks:

File: www/src/privacy.md [hunk 2]
```diff
@@ -1,21 +1,41 @@
...
+
+This chatmail server neither asks for nor retains personal information. 
+Chatmail servers exist to reliably transmit (store and deliver) end-to-end encrypted messages
+between user's devices running the Delta Chat messenger app. 
+Technically, you may think of a Chatmail server as 
+an end-to-end encrypted "messaging router" at Internet-scale. 
+
+A chatmail server is very unlike classic e-mail servers (for example Google Mail servers)
+that ask for personal data and permanently store messages. 
+A chatmail server behaves more like the Signal messaging server
```
[the remark this answers is not in the document]
Comment (hpk42) [reply www/src/privacy.md:22]: On a scale of "signal server" to "Gmail", chatmail servers are much closer to how signal operates than Gmail.  I think it's fine to say that.

[the remark this answers is not in the document]
Comment (missytake) [reply www/src/privacy.md:22]: True, and this is the privacy policy, not something very public-facing. It's fine if some metaphor we use requires technical understanding.

**Pull request merged**